src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
+@import './utils/variable';
```

Comment:
The import path should not include the file extension unless it is necessary. You can omit the '.scss' extension when importing SCSS files.